### PR TITLE
Piecestore proto change

### PR DESCRIPTION
**Pull request by aleitner**
* Add action and storage node id to `RenterBandwidthAllocation`
* Renamed payer and renter to satellite_id and uplink_id in `PayerBandwidthAllocation`

### CODE REVIEW

File: pkg/audit/verifier.go [hunk 1]
```diff
@@ -56,13 +56,13 @@ func NewVerifier(transport transport.Client, overlay overlay.Client, id provider
 	return &Verifier{downloader: newDefaultDownloader(transport, overlay, id)}
 }
 
-func (d *defaultDownloader) dial(ctx context.Context, node *pb.Node) (ps client.PSClient, err error) {
+func (d *defaultDownloader) dial(ctx context.Context, storagenode *pb.Node) (ps client.PSClient, err error) {
```
Comment (mniewrzal) [inline pkg/audit/verifier.go]: can you change it to `storageNode`?

Comment (aleitner) [reply pkg/audit/verifier.go]: done

File: pkg/pb/piecestore.proto [hunk 2]
```diff
@@ -18,26 +18,29 @@ service PieceStoreRoutes {
...
 
-message RenterBandwidthAllocation {
+message RenterBandwidthAllocation { // Renter refers to uplink
   message Data {
-    PayerBandwidthAllocation payer_allocation = 1;
-    int64 total = 2;
+    PayerBandwidthAllocation payer_allocation = 1; // Bandwidth Allocation from Satellite
+    int64 total = 2;                               // Total Bytes Stored
+    bytes storage_node_id = 3;                     // Storage Node Identity
+    string action = 4;                             // GET or PUT
```
Comment (jtolds) [inline pkg/pb/piecestore.proto]: i think the action needs to go in the payer allocation

File: examples/piecestore-client/main.go [hunk 3]
```diff
@@ -91,7 +95,20 @@ func main() {
 
 			id := client.NewPieceID()
 
-			if err := psClient.Put(context.Background(), id, dataSection, ttl, &pb.PayerBandwidthAllocation{}, nil); err != nil {
+			allocationData := &pb.PayerBandwidthAllocation_Data{
+				Action: "PUT",
```
Comment (egonelbre) [inline examples/piecestore-client/main.go]: Lets use a constant enum here instead.

Comment (aleitner) [reply examples/piecestore-client/main.go]: done

**Pull request merged**